src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalPrice = 40;
+  const discountForSevenDays = 50;
+  const discountForThreeDays = 20;
+  const rentalDays = days;
+  const totalCost = dailyRentalPrice * rentalDays;
+  let finalCost;
```

Comment:
Consider using 'const' instead of 'let' for 'finalCost' since its value is not reassigned after initialization. This aligns with checklist item 3.